### ISSUE TITLE
Usage not clear

### ISSUE THREAD

**Issue by Vishnu-C** [mMerlin/stl2scad#1]
**Usage of this script is not clear.**
Command to run the .stl file in testfiles would be a nice example to demonstrate the usage of this script.

**Comment by mMerlin** [comment]
Yep. That was left out of the notes. Been a (long) while since I looked at this. I do not currently have an environment setup to do a proper run through but:

`openscad test01.scad` displays a couple of tetrahedron objects created by tetrahedron.scad. This was a proof of concept, to make sure that openscad was setup to work with something simple, before blaming the conversion code on display failures.

```sh
../stl2scad test01.stl
```

Should (cant run it right now) generate files `openscad_model_001.scad` and `openscad_model_002.scad`

```sh
openscad test01a.scad
```

should display the converted model.

I need to look at this again, and (at least) modernize it to use (only) python3. Also switch to using pipenv for dependencies.

**Comment by Vishnu-C** [comment]
Thanks for the reply!

I confirm , 
`python ..\stl2scad.py test01.stl`

Doesn't perform conversion. Will wait for the code to work.

**Comment by mMerlin** [comment]
I [re] created my development environment for this.

I ran the code and documentation through some lint processing. This was from one of my very early efforts with python. I found that the line that actually wrote the generated files to disk had been commented out while doing some debugging, and left that way. It should work now. The readme has been updated to show how to use the test files.

This is not really 'production ready'. Plenty more features planned in the notes, but this got off my priority list. It works (did work) for the cases I tried it on. The next part was to start doing object extraction from the point, face, edge, surface data. Which is a lot harder than just using a polyhedron.

EDIT: When I wrote this, OpenSCAD did not have an stl import. Now it does. I do not know how much better or worse this is. It is a quite simple conversion to polyhedron, so not likely to be much better. Other than maybe the duplication removal that was the reason I started this in the first place.
